system/indy-node-tests/TestMultiSigSuite.py

```diff
@@ -43,8 +43,9 @@ class TestMultiSigSuite:
         trustee_did, _ = get_default_trustee
         new_did, new_vk = await did.create_and_store_my_did(wallet_handler, '{}')
         req = await ledger.build_nym_request(trustee_did, new_did, new_vk, random_string(5), None)
-        req = await ledger.sign_request(wallet_handler, trustee_did, req)
+        # if order of sign and multisign will be changed - test fails
```

Comment:
Don't you think that the test is ready for activation (I mean removing `skip` mark)?

Reply:
Removed.